docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -248,6 +247,7 @@ For the `s3` backend, the following additional properties are supported in the `
 - `disable_aws_client_checksums`: When `true`, disable computing and checking checksums on the request and response,
   such as the CRC32 check for DynamoDB. This can be used to workaround
   https://github.com/gruntwork-io/terragrunt/issues/1059.
+- `accesslogging_bucket_name`: (Optional) When provided as a valid `string`, the S3 bucket that is created to store the terraform state will create an additional S3 bucket as a `target bucket` for the logs output. If not provided, or sting is empty or invalid S3 bucket name, then server access logging for the S3 bucket storing the terraform state will be disabled.
```

Comment:
```suggestion
- `accesslogging_bucket_name`: (Optional) When provided as a valid `string`, create an S3 bucket with this name to store the access logs for the S3 bucket used to store Terraform state. If not provided, or string is empty or invalid S3 bucket name, then server access logging for the S3 bucket storing the terraform state will be disabled.
```

Also, shouldn't empty string or invalid bucket name or any other invalid input produce an error, rather than just silently disabling access logging?

Reply:
So any invalid input for an S3 bucket name will produce an error from the S3 client: `InvalidBucketName: The specified bucket is not valid.`.

I've decided to leave the case where "empty" because, I didn't know how to differentiate between `""` and `nil` for strings - they seem to be treated the same coming from the config. 

One option is as you've suggested to treat these two cases separately (_maybe with checking the length of the string passed_), and/or update the logs - and be specific why the logging was disabled.